driver/utils.go

```diff
@@ -16,7 +19,22 @@ var (
 	POD_VOL_LOCATOR  = "/volume"
 )
 
-func getAPIClient(secrets map[string]string, apiURL string) (*quobyte.QuobyteClient, error) {
+var clientCache *cache.Cache = nil
+
+func initClientCache() {
+	if clientCache == nil {
+		var err error
+		clientCache, err = cache.New(1000)
+		if err != nil {
+			klog.Fatalf("Could not initialize client cache")
+		}
+	}
+}
+
+func getAPIClient(secrets map[string]string, apiURL *url.URL) (*quobyte.QuobyteClient, error) {
+	if clientCache == nil {
```

Comment:
Nitpick, we check two times in succession `if clientCache == nil`, before and after calling `initClientCache`

Reply:
Added as a precaution. Nil pointers can result in fatal errors.
Checking twice does not do much harm here.